src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/StageReleaseContentFunction.cs

```diff
@@ -53,7 +53,7 @@ public class StageReleaseContentFunction
                 }).GetNextOccurrence(DateTime.UtcNow);
                 var context = new PublishContext(nextScheduledPublishingTime, true);
                 await _contentService.UpdateContent(context, message.Releases.Select(tuple => tuple.ReleaseId).ToArray());
-                await UpdateContentStage(message, Complete);
+                await UpdateContentStage(message, Scheduled);
```

Comment:
Hey, not 100% sure here, but I guess CompletePublishingIfAllPriorStagesComplete *is* now safe to call by the files function at any time as it will explicitly check that Content and Files have both been marked as Complete before it attempts to do anything else with them.